packages/launch/src/focused-launch/success/index.tsx

```diff
@@ -82,14 +96,16 @@ const Success: React.FunctionComponent = () => {
 	const copyButtonLabelIdle = __( 'Copy Link', __i18n_text_domain__ );
 	const copyButtonLabelActivated = __( 'Copied!', __i18n_text_domain__ );
 
+	const isLaunchComplete = ! isSiteLaunching && isPostSaved;
```

Comment:
This is fixing a bug where clicking the "Back Home' button in Success view would trigger a browser alert if there were unsaved changes on the post.